components/kubeflow/launcher/src/train.template.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: kubeflow.org/v1alpha2
+apiVersion: kubeflow.org/v1
```

Comment:
TFjob cannot works if you only update here, some fields of tfjob CRD update after v1alpha2
Refer to #1494 
I can re-pick up #1494 for v1 if you hope me do that